enos/enos.py

```diff
@@ -117,6 +119,10 @@ def up(env=None, **kwargs):
     base_inventory = seekpath(env['config']['inventory'])
     inventory = os.path.join(env['resultdir'], 'multinode')
     generate_inventory(env['rsc'], base_inventory, inventory)
+
+    if kwargs['--skip-install']:
```

Comment:
@msimonin I guess I should have explained my use case first. For our purposes, we would like to make use of enos _just_ to bring up and teardown bare-metal machines on chameleon